src/GitVersionCore/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using GitVersion.Common;
+
+namespace GitVersion.Extensions
+{
+    using Microsoft.Extensions.DependencyInjection;
```

Comment:
I can't quite remember what the standard for where to place `using` statements is within the codebase, but I think we should be consistently placing them outside **or** inside the `namespace` clause, not both.

Reply:
I changed to be outside, to be consistent